migration-tool/src/test/java/software/amazon/awssdk/migration/recipe/UpgradeSdkDependenciesTest.java

```diff
@@ -85,7 +85,6 @@ private static String getVersion() throws IOException {
     @Test
```

Comment:
I tried to add unit tests for gradle but the testing framework requires gradle to run it :(

Reply:
We could just invoke `gradle` from the unit test! (Gross, but it would work, right?)